### PR TITLE
HTML template for notifications with FlowDock

### PR DESCRIPTION
**Pull request by t0mk**
I created an HTML template, similar to the existing email body template. I use it for content in the Flowdock message. It's rather generic, and I think it can be used also in other HTML-capable notification receivers like Hipchat.

In Flowdock it looks like:

### CODE REVIEW

File: manager/notifier.go [hunk 1]
```diff
@@ -55,10 +56,26 @@ Grouping labels:
...
 {{range $label, $value := .Alert.Payload}}
   {{$label}} = "{{$value}}"{{end}}`))
 
+var contentTmpl = htmltemplate.Must(htmltemplate.New("content").Parse(
+	`<p><b><i>{{.Alert.Description}}</i></b></p>
+
+<div><i>Grouping labels</i></div>
+<ul>
+{{range $label, $value := .Alert.Labels}}
+<li><b>{{$label}}:</b> {{$value}}</li>
```
Comment (juliusv) [inline manager/notifier.go]: Indent the `li` within the `ul`?

File: manager/notifier.go [hunk 2]
```diff
@@ -370,17 +390,22 @@ func newFlowdockMessage(op notificationOp, config *pb.FlowdockConfig, a *Alert)
 	case notificationOpResolve:
 		status = "resolved"
 	}
+	contentBuf := new(bytes.Buffer)
```
Comment (juliusv) [inline manager/notifier.go]: Style nit: we only use `new` or `make` when required. Instead use `contentBuf := &bytes.Buffer{}` here (or `var contentBuf bytes.Buffer` and then `&contentBuf` as the argument).

**Comment by juliusv** [comment]
:+1: otherwise. We will want to migrate this to a generalized and user-configurable templating system soon.

**Comment by juliusv** [comment]
@t0mk Sorry, this one slipped through the cracks due to the lack of another ping after the last update. Could you squash the commits (and rebase on current master if necessary).

**Comment by t0mk** [comment]
Yeah, I'll do it on Monday.

**Comment by t0mk** [comment]
uh, let me fix the test

**Comment by t0mk** [comment]
I squashed the commits, fixed the test, and wrote a test for flowdock notification.

**Comment by juliusv** [comment]
@t0mk Thanks! Sorry again - was on vacation :) Now finally merging.

**Pull request merged**